### PR TITLE
Updated snapd/snapctl to snapteld/snaptel in doc and large tests

### PR DESCRIPTION
**Pull request by andrzej-k**
Summary of changes:
- Updated ```snapd``` to ```snapteld``` and ```snapctl``` to ```snaptel``` in documentation
- Updated large tests to use new binaries (```snapteld``` and ```snaptel```)

How to verify it:
- Run ```examples/tasks/run-mock-meminfo.sh```
- Read the documentation

Testing done:
- Ran ```examples/tasks/run-mock-meminfo.sh```
- Ran large test

### CODE REVIEW

File: examples/tasks/task-mem.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       },
       "publish": [
         {
-          "plugin_name": "file",
+          "plugin_name": "mock-file",
```
Comment (katarzyna-z) [inline examples/tasks/task-mem.json]: Why did you change it to `mock-file`?

Comment (andrzej-k) [inline examples/tasks/task-mem.json]: That's because plugin being loaded in the test is "snap-plugin-publisher-mock-file" (examples/tasks/mock-meminfo.sh:30). Without this change tests were failing.

**Comment by nanliu** [comment]
@andrzej-k we should revisit this PR because this plugin is using a custom container (not intelsdi/snap) and the large test is not failing properly. See #30 for more info.
